common/models/config-file.js

```diff
@@ -211,7 +211,8 @@ module.exports = function(ConfigFile) {
     });
 
     function find(pattern, cb) {
-      glob(pattern, { cwd: workspaceDir }, cb);
+      // set strict to false to avoid perm issues
```

Comment:
icic, let's add that in the comment, otherwise LGTM

// set strict to false to avoid perm issues on windows 10, may affect all platforms